File: src/Envirotrax.App/envirotrax.app.client/src/app/shared/models/fog/fog-disposal-site-candidate.ts
import { PhysicalType } from './fog-disposal-site-enums';

export interface FogDisposalSiteCandidate {
    id?: number;
    name?: string;
    registrationNumber?: string;
    county?: string;
    physicalType?: PhysicalType;
    isActive?: boolean;
}

File: src/Envirotrax.App/Envirotrax.App.Server/Data/Models/Fog/FogDisposalSiteCandidate.cs
namespace Envirotrax.App.Server.Data.Models.Fog;

public class FogDisposalSiteCandidate
{
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public string RegistrationNumber { get; set; } = null!;
    public string County { get; set; } = null!;
    public PhysicalType PhysicalType { get; set; }
    public bool IsActive { get; set; }
}
